data/membership-schema/src/main/resources/net/corda/schema/membership/member/static/registration/1.0/corda.member.static.registration.json

```diff
@@ -4,13 +4,39 @@
   "title": "Corda static member registration context schema",
   "description": "Definition of the registration context required for registering a member in a static group.",
   "type": "object",
+  "patternProperties": {
+    "^corda.roles.[0-9]+$": {
+      "description": "The member role. Can be multiple specified by index.",
+      "type": "string",
+      "enum": ["notary", "member"],
```

Comment:
What is the 'member' role in this case?

Reply:
Don't we need a role for a "Vanilla" member?

Reply:
I was thinking no role is a standard member. Then adding a role give additional capabilities. A notary member should be able to do everything a regular member can do with some additional functionality.